USE_CASES.md

```diff
@@ -1352,33 +1354,33 @@ To view Email Statistics via the API:
 package main
 
 import (
-	"fmt"
-	"log"
-	"os"
+  "fmt"
```

Comment:
Please revert these indentation examples, the current version is from `go fmt` which is our standard for formatting.